angular-intro-project/src/app/course-item/course-item.component.ts

```diff
@@ -8,8 +9,14 @@ import { Component, OnInit, Input } from '@angular/core';
 export class CourseItemComponent implements OnInit {
 
   constructor() { }
-  @Input() item;
+  @Input() item: Course;
+  @Output() itemDeleted = new EventEmitter<boolean>();
+
   ngOnInit() {
   }
 
+  deleteItem(id) {
```

Comment:
Let's use return types for methods and types for parameters. You can update _tslint.json_ `typedef` section for this.